gdx/src/com/badlogic/gdx/utils/Queue.java

```diff
@@ -245,19 +255,22 @@ public T removeIndex (int index) {
 		return value;
 	}
 
-	/** Returns true if the queue has one or more items. */
+	/** Returns true if the queue has one or more items.
+	 * @return true if not empty, else false */
```

Comment:
no need to repeat yourself: "@return true if the queue has one or more items." is enough. You can simplify at other places too.

Reply:
I changed the text to not duplicate the return tag, but to provide information for the function. Without a body, the javaDoc might look strange when observing the function.

Reply:
does it add any value? no. IMO javadoc is not even necessary for such self-explanatory methods.

Reply:
It is removed 👍